src/scopes/communicationLog/region.js

```diff
@@ -0,0 +1,31 @@
+import { Op } from 'sequelize';
+import { sequelize } from '../../models';
+
+const constructLiteral = (regions) => {
+  const sql = `
+    (SELECT DISTINCT "communicationLogId"
+     FROM "CommunicationLogRecipients" clr
+     JOIN "Recipients" r ON clr."recipientId" = r."id"
+     JOIN "Grants" gr ON r.id = gr."recipientId"
+     WHERE r."deleted" = false
+     AND gr."regionId" IN (${regions.map((regionId) => sequelize.escape(regionId)).join(',')}))
+  `;
```

Comment:
this works fine; however, is there a region you don't just access the regionId in the data blob?

Reply:
agree tho is we have the region in the blob why not use that.

Reply:
e82c7cb61c2eb78231e11ac2d4645ee1f481230d :tada: